addons/website_sale_loyalty/static/src/js/coupon_toaster_widget.js

```diff
@@ -11,24 +11,24 @@ const CouponToasterWidget = publicWidget.Widget.extend({
 
     start() {
         let options = {};
-        const $content = this.$('.coupon-message-content');
-        const $title = this.$('.coupon-message-title');
+        const contentEl = this.el.querySelector(".coupon-message-content");
+        const titleEl = this.el.querySelector(".coupon-message-title");
         let message = null;
 
-        if ($content.length) {
-            message = $content[0].innerHTML;
-            if ($title.length) {
-                Object.assign(options, {title: $title[0].innerHTML});
+        if (contentEl.length) {
+            message = contentEl.innerHTML;
+            if (titleEl.length) {
+                Object.assign(options, { title: titleEl.innerHTML });
             }
-        } else if ($title.length) {
-            message = $title[0].innerHTML;
+        } else if (titleEl.length) {
```

Comment:
I do not think `contentEl.length` and `titleEl.length` return what you expect.

Reply:
Yes, you are rigth. I think it skipped my mind while converting :sweat_smile:.

Reply:
@bso-odoo Done changes :+1: